### PR TITLE
Fix (Core\DB) Hostil to Hostile spelling fix

### PR DESCRIPTION
**Pull request by acidmanifesto**
## Changes Proposed:
-  Corrects Hostil to Hostile

## Issues Addressed:

- Closes 

## SOURCE:

Auto-Correct\Spell Check\Webster Dictionary Online

## Tests Performed:

- Builds
- Performs

## How to Test the Changes:

1. spelling correction. no need to really test but does effect the .debug hostil gm cmd that is now .debug hostile

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/scripts/Commands/cs_debug.cpp [hunk 1]
```diff
@@ -78,7 +78,7 @@ class debug_commandscript : public CommandScript
         {
             { "setbit",         HandleDebugSet32BitCommand,            SEC_ADMINISTRATOR, Console::No },
             { "threat",         HandleDebugThreatListCommand,          SEC_ADMINISTRATOR, Console::No },
-            { "hostil",         HandleDebugHostileRefListCommand,      SEC_ADMINISTRATOR, Console::No },
+            { "hostile",         HandleDebugHostileRefListCommand,      SEC_ADMINISTRATOR, Console::No },
```
Comment (Si1ker) [inline src/server/scripts/Commands/cs_debug.cpp:81]: align as the others in the list.

**Comment by Si1ker** [comment]
Also, you could expand this PR to any other file affected by the typo
see https://github.com/azerothcore/azerothcore-wotlk/search?p=1&q=Hostil

File: src/server/scripts/Commands/cs_debug.cpp (hunk 1 above)
Comment (acidmanifesto) [reply src/server/scripts/Commands/cs_debug.cpp]: got it. added in two more extra spaces.

File: src/server/scripts/Commands/cs_debug.cpp [hunk 2]
```diff
@@ -78,7 +78,7 @@ class debug_commandscript : public CommandScript
         {
             { "setbit",         HandleDebugSet32BitCommand,            SEC_ADMINISTRATOR, Console::No },
             { "threat",         HandleDebugThreatListCommand,          SEC_ADMINISTRATOR, Console::No },
-            { "hostil",         HandleDebugHostileRefListCommand,      SEC_ADMINISTRATOR, Console::No },
+            { "hostile",           HandleDebugHostileRefListCommand,      SEC_ADMINISTRATOR, Console::No },
             { "anim",           HandleDebugAnimCommand,                SEC_ADMINISTRATOR, Console::No },
             { "arena",          HandleDebugArenaCommand,               SEC_ADMINISTRATOR, Console::No },
             { "bg",             HandleDebugBattlegroundCommand,        SEC_ADMINISTRATOR, Console::No },
```
Comment (Si1ker) [inline src/server/scripts/Commands/cs_debug.cpp:84]:
```suggestion
            { "hostile",        HandleDebugHostileRefListCommand,      SEC_ADMINISTRATOR, Console::No },
            { "anim",           HandleDebugAnimCommand,                SEC_ADMINISTRATOR, Console::No },
            { "arena",          HandleDebugArenaCommand,               SEC_ADMINISTRATOR, Console::No },
            { "bg",             HandleDebugBattlegroundCommand,        SEC_ADMINISTRATOR, Console::No },
```

**Review by Si1ker — CHANGES_REQUESTED** [review]
indentation and alignment

**Comment by acidmanifesto** [comment]
I was just messing around. But good catch on the other stuff. Ill check it out when im back home

**Comment by acidmanifesto** [comment]
@Si1ker lets see how this goes with the tests XD

**Review by Si1ker — APPROVED** [review]
Seems good now, no tests are needed for refactoring, hope a current staff member drops and approves for merge tagging, gj.

**Comment by Nyeriah** [comment]
Potato potata 🤷🏼‍♂️

**Review by Nyeriah — APPROVED** [review]

**Comment by Kitzunu** [comment]
🥔🍠

**Comment by acidmanifesto** [comment]
🍆

**Pull request merged**